requirements.txt

```diff
@@ -0,0 +1,66 @@
+addict==2.2.0
```

Comment:
@SamarthVP capture requirements with `pip freeze --local > requirements.txt`

Reply:
That's very useful